submissions/kyle/bonus-testing-part-one.js

```diff
@@ -50,12 +49,14 @@ var nextAnimal;
   /* TODO:
       Assign one of your favorite animals to nextAnimal using Math.random()
       Your code begins on the next line: */
-
+nextAnimal = favoriteAnimals[Math.floor(Math.random() * favoriteAnimals.length)];
   /* TODO:
       Write a test! Use the `expect()` function we defined earlier to ensure
       that an element in the favoriteAnimals array was assigned to nextAnimal.
       Remember to: pass in your expression, and write a failure and a success
       message. Your test begins on the next line: */
-
-
-
+expect(
+  nextAnimal === 'elephants',
+  'my test failed! The next animal should have been elephants, but currently is ' + nextAnimal,
```

Comment:
This is great! Think about how we could refactor this test to be able to pass no matter which `nextAnimal` is generated 😉